app/src/test/groovy/gov/hhs/cdc/trustedintermediary/ApplicationContextTest.groovy

```diff
@@ -0,0 +1,17 @@
+package gov.hhs.cdc.trustedintermediary.context
+
+import spock.lang.Specification
+
+class ApplicationContextTest extends Specification {
+    def "singleton object"() {
+        setup:
+        def contextA = ApplicationContext.getInstance()
+        def contextB = ApplicationContext.getInstance()
+
+        when:
+        def result = contextA == contextB?true:false
+
+        then:
+        result == true
+    }
```

Comment:
I want to add a test case to test the retrieval of the implementations. Currently, we have no classes to test this with, but I'm open to suggestions. Let me know your thoughts.

Reply:
You could test it with a standard Java class. For example,

```java
applicationContext.register(String.class, "DogCow");
```

Reply:
Yes! We think alike. I was thinking in those parameters. I'll update it when I'm able to, thanks!